### DISCUSSION TITLE
Spacing question

### DISCUSSION THREAD

**Discussion by catches** [daattali/beautiful-jekyll#981] [General]
Resolved, see below for adjusting margins between images with HTML.

**Comment by daattali** [comment]
This requires custom css. You'll need to take a simple CSS tutorial and add a CSS file

**Comment by catches** [comment]
Thanks for the quick response, I got it solved, what I did was instead just used a bit of HTML.

`<a href="Test"><img src="https://test.png" title="Test" style="margin:5px;" /></a>
<a href="Test"><img src="https://test.png" title="Test" style="margin:5px;" /></a>`
